src/Dashboard/Data/RecentInvocationEntry.cs

```diff
@@ -16,6 +16,7 @@ public class RecentInvocationEntry
         private const string StartTimeKey = "StartTime";
         private const string EndTimeKey = "EndTime";
         private const string SucceededKey = "Succeeded";
+        private const string HeartbeatExpritedTimeKey = "Heartbeat2Name";
```

Comment:
I don't think this is written to storage.  All this metadata stuff is legacy that looks like pure in-memory marshalling - not sure what value it's actually adding. But I tried to cooperate with it to keep this change scoped.